ea_coreg_fa.m

```diff
@@ -1,4 +1,18 @@
 function ea_coreg_fa(options,usebrainmask)
+% 
```

Comment:
Please provide a description of the function.

Reply:
Coregister FA image to anatomical anchor image.
Will export FA image from DTI if it doesn't exist yet.